_posts/2016-11-01-devfest-cfp-cfv.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Call for Papers and Volunteers"
+date:   2016-09-01 23:59:59
+isStaticPost: false
+---
+
+We hereby open our this years Call for Particpants!
+
+Do you want to talk about
+
+* Artificial Intelligence
+* Android
+* Cloud
+* Firebase
+* Go
+* Internet of Things
+* VR or
+* Web
+
+then please head over to our [Call for Participation](https://www.papercall.io/devfest-berlin-2016). 
+
+If you are on the other hand not the talking type, but want to support us with more manual labor, please consider filling out our [Call for Volunteers](http://bit.ly/devfest-berlin-volunteers).
```

Comment:
Suggestion: "If you are not the talking type, you can help out by signing up as volunteer. Check out the [Call for Volunteers](…)."